### PR TITLE
Removed NO MORE update in hide and show stats per #74

### PR DESCRIPTION
**Pull request by andmarek**
Co-authored-by: Eric Marcanio <ericmarcanio@gmail.com>
Co-authored-by: Matthew Phipps <mnphipps98@gmail.com>

Looks like there was no reason to update this.NO_MORE in hideStats() unless hideStats() is a misnomer.

### CODE REVIEW

**Comment by andmarek** [comment]
fixes #74

**Comment by samtheprogram** [comment]
- the NO_MORE prop should be removed, such that the ReceiverConnection manages the StatsHandler itself
- This breaks the Stats dropdown since NO_MORE is no longer triggering ReceiverConnection to open the StatsHandler -- so ReceiverConnection needs to set up StatsHandler regardless when the connection is created.
- Finally, when ReceiverConnection is unmounted, the StatsHandler needs to be destroyed.

**Comment by andmarek** [comment]
I deleted the NO_MORE data in Receiver and removed any trace of it.  I find that a bigger issue is that there is no onPeerStateChanged when a user who is viewing a stream clicks on the "2N" at the top and goes back to the home page.  This could root from the issue with the viewers not going down when a user leaves as well.  I believe this is why the stream data keeps getting sent, as the getStats() call only happens when the peer state is connected.

File: src/components/ReceiverConnection.vue [hunk 1]
```diff
@@ -227,6 +228,7 @@ export default {
         event.iceConnectionState === 'connected' &&
         event.signalingState === 'stable'
       ) {
+        console.log(event.iceConnectionState);
```
Comment (samtheprogram) [inline src/components/ReceiverConnection.vue]: This could be removed.

File: src/views/Receiver.vue [hunk 2]
```diff
@@ -80,6 +80,8 @@ video
 
 #volume-slider
   max-width: 120px
+  @supports (-webkit-touch-callout: none) // iOS volume slider doesn't work, so hide it
```
Comment (samtheprogram) [inline src/views/Receiver.vue]: This should be in a different PR / ticket... was this written in this PR or did it end up in this branch after pulling from master?

File: src/views/Receiver.vue [hunk 3]
```diff
@@ -217,8 +219,6 @@ export default {
       isPlaying: false,
       theaterMode: false,
       statsVisible: false,
-      volume: window.localStorage.getItem('volume') || 0.5,
```
Comment (samtheprogram) [inline src/views/Receiver.vue]: 🤔 this should definitely not been removed

File: src/views/Receiver.vue [hunk 4]
```diff
@@ -397,21 +394,18 @@ export default {
...
     },
     hideStats() {
       this.statsVisible = false;
-      this.NO_MORE = true;
     },
     bytesToSize(bytes) {
-      // TODO: Should this be 1024?
-      var k = 1000;
-      var sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB'];
+      let k = 1000;
```
Comment (samtheprogram) [inline src/views/Receiver.vue]: These could be const instead of let.

**Comment by samtheprogram** [comment]
These changes were rebased and applied in PR #133. Thanks! Closing PR.
